web_calendar_view_custom_colors/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_calendar_view_config,access_calendar_view_config,model_calendar_view_config,base.group_user,1,1,1,1
```

Comment:
Do not give access to everyone, see the already existing rights for calendar event to see who should be able to read and who should be acble to change the settings.